content/outcomes/application-development/cloud-native-development.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Cloud Native Development
+weight: 40
+layout: single
+---
+
+Cloud Native is at the heart of modern application development endeavors. Used to depict engineering practices and state of the art applications, the term "cloud-native" is omnipresent and often misunderstood. There is a good chance that anyone you ask will provide a different definition.
+
+![Cloud Native Development](/images/outcomes/app-dev-onboarding/cloud-native-development.jpg)
+
+## What you will learn
+
+In this article, you will learn to:
+
+- [Define one meaning of cloud native](#meaning)
+- [List three major outcomes of cloud native](#outcomes)
+- Explain the main cloud native concepts for ...
+	- [application architecture](#app-archi)
+	- [software life-cycle](#sw-life-cycle)
+	- [supporting infrastructure](#support-infra)
+	- [team culture](#culture)
+- [Identify the twelve factors of cloud native applications](#factors)
+
+------
+
+## Define one meaning of cloud native {id=meaning}
+
+>  “Cloud is not a place; it’s a way of doing IT” - Michael Dell
+
+There are many definitions for cloud native. The definition we use in this article is an approach to building and running applications that exploit the advantages of the cloud computing delivery model. At its heart, cloud native is structuring teams, culture, and technology to utilize automation and architectures to manage complexity and unlock velocity.
+
+Cloud native development, often referred to as modern application development, defines how applications are created and deployed. It does not define where the deployment occurs. For example, public cloud, private cloud, hybrid cloud, multi-cloud.
+
+## List three major outcomes of cloud native {id=outcomes}
+
+Organizations that adopt cloud native practices and technology generally report the following outcomes:
+
+1. More efficient and happier teams. Big problems are broken down into smaller pieces for more focused, efficient and responsive teams.
+2. Higher degree of automation, more determinism, less drudgery and less manual work. Infrastructure is self-healing, self-managed, and available as self-service.
+3. Increased non-functionalities for application landscape: improved reliability, portability, deep security, higher efficiency, and in-depth observability.
+
+## Explain the main cloud native concepts {id=concepts}
+
+Cloud native concepts are numerous. The following ones are the most common. They are sorted by their context:
+- [application architecture](#app-archi)
+- [software life-cycle](#sw-life-cycle)
+- [supporting infrastructure](#support-infra)
+- [team culture](#culture)
```

Comment:
Are we asking the reader to explain these four cloud native concepts in general, or the 18 total concepts contained within?

Reply:
These four beasts are flagged as "context" for the concepts. Indeed, here the 18 concepts are in focus.